ifrs17/Import/ImportStorage.ipynb

```diff
@@ -279,7 +280,8 @@
                 "\n        ",
                 "\n        //Variables",
                 "\n        var rawVariables = parsedRawVariables.Concat(openingRawVariables)",
-                "\n            .Concat(await querySource.Query<RawVariable>().Where(rv => rv.Partition == TargetPartition).Where(rv => primaryScopeFromLinkedReinsurance.Contains(rv.DataNode)).ToArrayAsync()); ",
+                "\n            .Concat(await querySource.Query<RawVariable>().Where(rv => rv.Partition == TargetPartition)",
+                "\n            .Where(rv => primaryScopeFromLinkedReinsurance.Contains(rv.DataNode)).ToArrayAsync()); ",
```

Comment:
Why do we add RawVariables from primaryScopeFromLinkedReinsurance but not from primaryScopeForPaaLrc even if they are in AddedPrimaryScope

Reply:
This is not part of the changes of this PR. 
The reason for it is that a PAA-LRC GIC might require calculation even though no new inputs are available. 
However, I think it is not correct as it could delete variable in some cases. It is safer to add those RawVariables anyway. 
Thanks for the observation.